ivy/functional/ivy/linear_algebra.py

```diff
@@ -758,21 +758,24 @@ def eigvalsh(
 
     >>> x = ivy.array([[[3.0,2.0],[2.0,3.0]]])
     >>> y = ivy.zeros([1,2])
-    >>> ivy.eigvalsh(x, out=y)
-    >>> print(y)
+    >>> ivy.eigvalsh(x)
```

Comment:
the point here was to demonstrate y being written to when out=y. the change does not reflect this.